test/runtests.jl

```diff
@@ -1446,3 +1446,64 @@ end
         @test predict(mdl1) ≈ predict(mdl2)
     end
 end
+
+@testset "dropcollinear with GLMs" begin
```

Comment:
where is `dropcollinear=false` tested?

Reply:
Thanks, @bkamins. I will add test cases accordingly.